Gemfile.lock

```diff
@@ -598,4 +599,4 @@ RUBY VERSION
    ruby 2.4.4p296
 
 BUNDLED WITH
-   1.17.3
+   2.0.1
```

Comment:
i dunno i must've updated recently. is it a problem?

Reply:
ugh, `You must use Bundler 2 or greater with this lockfile.`

Will try again downgrading bundler...